packages/react-renderer-demo/src/components/code-editor/index.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import { styled } from '@mui/material/styles';
 import Highlight, { defaultProps } from 'prism-react-renderer/';
 import ghTheme from 'prism-react-renderer/themes/github';
-import vsTheme from 'prism-react-renderer/themes/vsDark';
+import oceanicNext from 'prism-react-renderer/themes/oceanicNext';
```

Comment:
Is there any specific reason for changing the theme other than personal preference?

Reply:
umm, It's actually not my favorite one😢, but I think we should use a more grey background color in the code block, instead of a nearly pure black color as the code background, because we basically use low saturation colors across the whole site. A nearly pure black block area can make the site less harmonious. Besides oceanicNext, _dracula_ is also a good choice. The syntax highlight color of dracula is actually more suitable to the main theme color of the site.  That's just my thought, and I'm also totally OK if you still want to use the vsDark theme 😄

Reply:
OK, that makes sense! I'll change it back to vsDark~

Reply:
OK, that makes sense! I'll change it back to vsDark~